docs/src/components/tooltip.md

```
@@ -0,0 +1,58 @@
+    </VCTooltipProvider>
+</template>
+```
+
+For most apps, install `<VCTooltipProvider>` once at the root and wrap individual usages with `<VCTooltip>` / `<VCTooltipTrigger>` / `<VCTooltipContent>`.
+
+## Theme keys
+
+| Key | Default class | Notes |
+|---|---|---|
+| `trigger` | `vc-tooltip-trigger` | |
+| `content` | `vc-tooltip-content` | Bubble; supports `data-state="delayed-open|closed"` for animation. |
```

markdownlint-cli2 (0.22.1)

[warning] Table column count
Expected: 3; Actual: 4; Too many cells, extra data will be missing

(MD056, table-column-count)